packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -230,9 +230,9 @@ export default function PostOptionsMenu({
   };
   const video = advancedSettings?.find(({ title }) => title === 'Videos');
   const onToggleVideo = async () => {
-    await onToggleSettings(video.id, false);
+    await onToggleSettings(video.id, true);
     await showMessageAndRemovePost(`⛔️ Video content blocked`, postIndex, () =>
-      onToggleSettings(video.id, true),
+      onToggleSettings(video.id, false),
```

Comment:
I need to double check, but I am sure that it was correct before.
The default value in database is `true` when it is not set to anything, i.e. video content is allowed.
So if you disable video content, you set the `enabled` column in `feed_advanced_settings` table to `false`.

Reply:
Ah yes but onToggleSettings reverses what you pass afaik (from what i tested) it accepts a default state

Reply:
This value is the `defaultEnabledState` and should not be changed. Maybe that parameter should not even be there in the first place, I believe we had that for some experiment in the past but the value there should reflect what we have in our DB for this entity which is `true`. That is the reason why this was strongly-typed.

Reply:
The whole issue here is that, it is a toggle, and the post options menu does not reflect the correct state, so even when it says it should block the option, it would then enable it if you re-click it.

Reply:
It also didnt reflect right in the database though.
It added it as "true" because it didnt exist yet for this user.

Secondary the option disappears from the post options menu when false so think thats fine right?

Maybe we can sync on it today?

Reply:
> It added it as "true" because it didnt exist yet for this user.

That is the default state from the AdvancedSettings not necessarily for the user.

Reply:
Validated with Lee this was the right fix.
CC: @omBratteng

Reply:
Just to note: the issue was after clicking the post options menu to block Videos, then clicked the Undo, it would fail.